src/libraries/System.Drawing.Common/pkg/System.Drawing.Common.pkgproj

```diff
@@ -2,13 +2,14 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Drawing.Common.csproj">
-      <SupportedFramework>netcoreapp3.0</SupportedFramework>
+      <SupportedFramework>netcoreapp3.0;net461</SupportedFramework>
```

Comment:
I think you should add all netstandard2.0 compatible frameworks here since the package will look like it supports them (your targets are the only thing preventing that)